algorithms/C/strings/reverse a string with spaces using 2d arrays

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<string.h>
+int main()
+{
+    char s1[100][100],s[100];
+    int k=0,j=0,i,n;
+    printf("Enter the string: ");
+    gets(s);                                   // K DENOTES NUMBER OF WORDS
+    for(i=0;s[i]!='\0';i++)                    // J DENOTES WORD LENGTH
+    {
+        if(s[i]==' ')
```

Comment:
what about the case where there are multiple spaces in the string separating the words, consider looking into [`sscanf ()`](https://en.cppreference.com/w/c/io/fscanf) and see how you can integrate into this program to make your life and this program much more simpler :)